src/scripts/main.js

```diff
@@ -1 +1,15 @@
 'use strict';
+
+const list = document.querySelector('.gallery__list');
+const largeImage = document.querySelector('#largeImg');
+
+list.addEventListener('click', (action) => {
```

Comment:
please rename 'action' to 'event', because it's Event Listener

Reply:
I have error. 'event' is already declared in the upper scope. So i called 'e'.